api/Window.json

```diff
@@ -6107,7 +6107,7 @@
               "version_added": true
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "37"
```

Comment:
Thank you for this PR, Joe. Are these definitely 37, not `"≤37"`? The linked commit didn't help me answer this question

Reply:
I know it was in Chromium WebView for sure starting in 37. I don't know anything about non-Chromium WebView. I'm rusty what covers that?

Reply:
I think that `"≤37"` would probably be more appropriate, since we're not sure when it was implemented and that it _could_ be in pre-Chromium WebView?  Considering the age of the feature, I'd be surprised if it was only just implemented in WebView 37, and not a version like WebView for Android 4.4 (Blink 30).